packages/contracts-bedrock/test/L2/L2CrossDomainMessenger.t.sol

```diff
@@ -35,6 +35,8 @@ contract L2CrossDomainMessenger_Constructor_Test is L2CrossDomainMessenger_TestI
         assertEq(address(impl.OTHER_MESSENGER()), address(0));
         assertEq(address(impl.otherMessenger()), address(0));
         assertEq(address(impl.l1CrossDomainMessenger()), address(0));
+        assert(bytes(impl.version()).length > 0);
```

Comment:
devinai please use forge's `assertGt` instead of native solidity asserts in your diff